ci/build-all

```diff
@@ -19,3 +19,9 @@ mkdir build
 mkdir -p $INSTALL_PREFIX
 ./configure --prefix $INSTALL_PREFIX
 sudo make install
+cd build/librina
+sudo make check-functional
+cd ../..
+sudo make install
```

Comment:
why repeating `sudo make install`?

Reply:
That is a hang over from the previous Travis CI way of building this. It is now removed with the last commit.